databaseManager.js

```diff
@@ -68,25 +68,36 @@ const database = () => {
     const _tagger = getOrCreateUser(tagger);
     const toTag = getOrCreateUser(userToTag, 0, true);
 
-    _tagger.points += 1;
+    const bounty = _calculateBounty(toTag.lastTagged);
+
+    _tagger.points += bounty;
     if (_tagger.isTagged) {
       _tagger.isTagged = false;
     }
 
     // next, reset the tagged user's streak
     const taggedTime = dayjs();
     const streak = taggedTime.diff(toTag.lastTagged, "day");
+    const newPoints = toTag.points + bounty;
     users[userToTag.id] = {
       ...toTag,
       lastTagged: dayjs(),
       isTagged: true,
+      points: newPoints,
       ...{
         bestStreak: streak > toTag.bestStreak ? streak : toTag.bestStreak,
       },
       totalSurvivalTime: toTag.totalSurvivalTime + streak,
     };
     exportDb();
   };
+  const _calculateBounty = (lastTagged, isTagged = false) => {
```

Comment:
How do you see the `isTagged` param being used?

Reply:
i my thought is that calculate bounty by taking isTagged can be used in a loop without having to deal with splitting logic on whether the user is tagged or not.  I use it once in the scoreboard just so i can lazily calculate displayScore on all users instead of writing a ternary check or somethign if bounty should be added.  I think it also makes it more explicit how its expected to be used. In the event someone's old and doesn't know tagger's shouldn't have a bounty. 

I think there is probably a better way to do it, by have a getter on user so all instances of grabbing score could just be getScore(user) but that wasn't really the current pattern.